### ISSUE TITLE
Documents gone

### ISSUE THREAD

**Issue by novascreen** [classeur/classeur#144]
The second time now a document i was working on for multiple days has simply disappeared. I believe in both cases i've created a new document, kept editing it for a while and when i closed it it simply didn't exist anymore (hitting backspace fails, giving me a message "{ID} is not accessible"). I tried using the REST API to retrieve the document, but that also just gives me a 403. Is there any possibility these documents still exist somewhere? The first one i've made my peace with and i don't think i have the id anymore, but the one from today would be really great to get back. I love Classeur for writing, but this is kind of infuriating.

**Comment by benweet** [comment]
Hi @novascreen 
Sorry to hear that.
Do you have any error in the javascript console?
You can send us your file ID by email, we'll try to figure out what happened.
Thanks

**Comment by novascreen** [comment]
About 1000 times:

```
app-min.js:formatted:43968 Uncaught TypeError: Cannot set property 'state' of undefined
```

i'll send the id via email, but looking at this it might have never synced to begin with. There is nothing left in localstorage either.

**Comment by novascreen** [comment]
Lines 43968 and following:

```
            t.currentFile.content.state = {
                selectionStart: i.cledit.selectionMgr.selectionStart,
                selectionEnd: i.cledit.selectionMgr.selectionEnd,
                scrollPosition: i.getScrollPosition() || t.currentFile.content.state.scrollPosition
```

**Comment by benweet** [comment]
Yeah, I'm afraid the file didn't reached the server and was probably never written to the local storage. Looks like a tricky one, let us know if you have a way to reproduce it.

**Comment by novascreen** [comment]
:cry: i'll let you know if i find a way to reproduce it. I was using the Chrome app.
The other factor could be that it was open for many days, is it possible that something broke when you pushed a new version? Maybe having open multiple windows?
I'll certainly play around with it a bit.

**Comment by novascreen** [comment]
Actually the multiple windows seem to cause the issue, i can reproduce the JS error.
- open a document in one window
- open a new window
- create a new file there
- start typing
- see JS errors in console / no new entry in localstorage

**Comment by benweet** [comment]
Yep just reproduced it! Investigating... Thanks very much for that.

**Comment by buckle2000** [comment]
Since there is a lot of issues about "multiple windows", is it good to prevent login the same account with two session? If someone do so, close the first session (force disconnect and show an error message).

**Comment by benweet** [comment]
Should be fixed in v1.7.0.
